### ISSUE TITLE
Ability to use response files and in-built parser (text => args) from spectre.console to provide consistent behavior

### ISSUE THREAD

**Issue by rajasekarshanmugam** [spectreconsole/spectre.console#580]
**Is your feature request related to a problem? Please describe.**
For some CLI, no of commands and the commands supported will be many. Using a typical command line is harder and has to be backed by other shell mechanisms. Would be good if a standard response file is supported. Response files typically are simple text files containing the commands, options/arguments. Response files come in different flavors too, single command/option per line or as a single line etc.

**Describe the solution you'd like**
Response files are typically passed via command-line argument @responsefile and this could be the only argument when passed. Console programs tend to read the response file instead of the command line arguments and would parse/split as per needs instead of the default command line arguments (args[] via Main, or Environment.GetCommandLineArgs()).

**Describe alternatives you've considered**
Currently no alternative solution.  While it's easy to detect the presence of a response file argument parsing the text file into arguments is needed.

**Additional context**
-

**Comment by Simonl9l** [comment]
I'd Thumbs up this request,[ Nate McMasters](https://natemcmaster.github.io/CommandLineUtils/docs/response-file-parsing.html?tabs=using-attributes) has this in his CLI, but we have other reasons with continue with this one.

**Comment by oconnor0** [comment]
I thought I had read somewhere that Spectre.Console supported response files (to pass/intersperse command line arguments), but it looks from this that I must have misread?

**Comment by phil-scott-78** [comment]
Yeah, this feature request is still open. Happy to take a look at a pull request if someone wants to give it a go.
